i18n/locales/en.json

```diff
@@ -17,6 +17,7 @@
     },
     "hamburgerMenu": {
         "about": "About Find a Doc, Japan",
+        "balancesheet": "Balance Sheet",
```

Comment:
Would be cool if you could automate adding this to the other files

Reply:
haha, if only there was a way.....